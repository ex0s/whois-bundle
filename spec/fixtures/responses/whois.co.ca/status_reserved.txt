Domain is not available or is reserved by the registry.

This information is collected for information purposes only, and is intended
only for determining the owner of .CO.CA domains.  No guarantees will be made
regarding the accuracy of this data.
TERMS OF USE: By submitting this query you are agreeing that under no
circumstances will you use this any of this data:
(1) In any fashion relating to, supporting, or allowing the tranmission of any
form of mass unsolicited contact of any kind.
(2) Any form of high-volume automated or bulk lookups of any kind.
Storage or distribution of any of this information in any volume fashion
We reserve the right to terminate your access at any time.
