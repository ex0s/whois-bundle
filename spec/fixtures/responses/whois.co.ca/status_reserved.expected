#status
  %s == :reserved

#available?
  %s == false

#registered?
  %s == true


#reserved?
  %s == true
